README.md

```diff
@@ -46,8 +46,12 @@ The `notify` function can either be passed a string or an object.  When passing
 * `classes` - Optional. A list of custom CSS classes to apply to the message element.
 * `messageTemplate` - Optional. A string containing any valid Angular HTML which will be shown instead of the regular `message` text. The string must contain one root element like all valid Angular HTML templates (so wrap everything in a `<span>`).
 * `scope` - Optional.  A valid Angular scope object.  The scope of the template will be created by calling `$new()` on this scope.
-* `position` - Optional.  Currently `center` and `right` are the only acceptable values.
+* `position` - Optional.  'left', 'right' and 'center' are the only acceptable values.
+* `duration` - Optional. The duration (in milliseconds) of the message. A duration of 0 will prevent the message from closing automatically.
 * `container` - Optional.  Element that contains each notification.  Defaults to `document.body`.
+* `onOpen` - Optional. Function that is called when the notification is shown to the user. The function receives the message of the notification as its only parameter.
```

Comment:
I think all 3 events need to take a better argument than the message string.  I would think the actual return value (which is an object that contains a modifiable message property as well as a close method) would be more appropriate.